src/evergreen.ts

```diff
@@ -118,6 +118,19 @@ export class client {
     return this.getUIResource(resource, params);
   }
 
+  /**
+   * Gets patches for a particular project
+   * @param projectName - Evergreen project name
+   * @returns a promise for the caller to handle responses
+   */
+  public getProjectPatches(projectName?: string, page?: number): AxiosPromise<models.Patches> {
+    const resource = "json/patches/project/" + projectName
```

Comment:
nit: prefer the use of template literals over string concatenation, i.e. `json/patches/project/${projectName}`